### ISSUE TITLE
The new Golang implementation for the BumpMinSdkFloor action fails when manfiest doesn't have namespace

### ISSUE THREAD

**Issue by Bencodes** [bazelbuild/rules_android#207]
The new implementation fails against our project when attempting to bump the min SDK value of a manifest that does not have the default namespace. We encountered this edge case in one of the Jetpack Compose libraries. Example manifest which was taken from the `androidx_compose_ui_ui_geometry` library:

```xml
<manifest package="androidx.compose.ui.anchor"></manifest>
```

The error:
```
2024/02/26 17:12:24 109 <nil>
Error modifying minSdkVersion: translating attribute name "uses-sdk" failed, got: unknown namespace: android
```

**Comment by jylinv0** [comment]
Fixed by acaf27a.

**Comment by Bencodes** [comment]
Thanks for the quick turn around @jylinv0!
